experimental/framework/tree-react-api/src/test/reactSharedTreeView.spec.ts

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { SchemaFactory, TreeConfiguration } from "@fluidframework/tree";
+import type { ContainerSchema } from "@fluidframework/fluid-static";
+import { treeDataObject } from "../reactSharedTreeView.js";
```

Comment:
This is erroring with:

> Output file '/home/craig/Work/FluidFramework.git/proxy/experimental/framework/tree-react-api/lib/reactSharedTreeView.d.ts' has not been built from source file '/home/craig/Work/FluidFramework.git/proxy/experimental/framework/tree-react-api/src/reactSharedTreeView.tsx'.ts(6305)

But I can't figure out why. The project reference from test to non-test looks correct and I'm not sure what else to check.